frontend/src/i18n/messages/en/index.ts

```diff
@@ -494,13 +550,6 @@ export const messages = {
     button: "Sign in with Login.gov",
     close: "Cancel",
   },
-  Hero: {
```

Comment:
Looks like there is some more that can be deleted here - most of the `index` section (other than page title and meta description) is now unused. Sorry I didn't catch that before

Reply:
I gotchu!! Sorry about that

Reply:
done!

Reply:
@doug-s-nava done!